hil/cli.py

```diff
@@ -797,6 +797,27 @@ def create_admin_user(username, password):
 
 
 @cmd
+def list_active_extensions(ext_type):
+    """List active extensions by type.
+
+    Default types: auth, obm, switches, network_allocators.
+
+    Use the 'all' argument to list all active extensions.
+    """
+    if not config.cfg.has_section('extensions'):
+        sys.exit("There are no active extensions.")
+
+    all_extensions = C.extensions.list_active()
+    if ext_type == 'all':
+        active = [ext[0] for ext in all_extensions]
+        sys.stdout.write('All active extensions: ' + ', '.join(active) + '\n')
```

Comment:
I would print each on its own line (and without the description), to make the output usable with e.g. grep.